nixos/modules/virtualisation/amazon-image.nix

```diff
@@ -64,20 +53,17 @@ in
     boot.extraModulePackages = [
       config.boot.kernelPackages.ena
     ];
-    boot.initrd.kernelModules = [ "xen-blkfront" "xen-netfront" ];
-    boot.initrd.availableKernelModules = [ "ixgbevf" "ena" "nvme" ];
```

Comment:
Does dropping `ena` from here affect the assertion on line 29? (`"ENA driver fails to build..."`)